src/components/Form.stories.js

```diff
@@ -0,0 +1,12 @@
+import Form from './Form.js';
+
+export default {
+  title: 'components/Form',
+  component: Form,
+  argTypes: {},
```

Comment:
you could set an onSubmit: 'onSubmit', here, if you wanted to test the functionality in storybook (but I think that's not that important)